crates/ruff/src/rules/flake8_simplify/snapshots/ruff__rules__flake8_simplify__tests__SIM223_SIM223.py.snap

```diff
@@ -116,4 +116,242 @@ SIM223.py:25:4: SIM223 [*] Use `False` instead of `... and False`
 27 27 | 
 28 28 | 
 
+SIM223.py:42:4: SIM223 [*] Use `False` instead of `... and False`
+   |
+42 | if a and "" and False:  # SIM223
+   |    ^^^^^^^^^^^^^^^^^^ SIM223
+43 |     pass
+   |
+   = help: Replace with `False`
+
+ℹ Suggested fix
+39 39 |     pass
+40 40 | 
+41 41 | 
+42    |-if a and "" and False:  # SIM223
+   42 |+if False:  # SIM223
+43 43 |     pass
+44 44 | 
+45 45 | if a and "foo" and False and "bar":  # SIM223
+
+SIM223.py:45:15: SIM223 [*] Use `False` instead of `... and False`
+   |
+45 |     pass
+46 | 
+47 | if a and "foo" and False and "bar":  # SIM223
+   |               ^^^^^^^^^^^^^^^^^^^^ SIM223
+48 |     pass
+   |
+   = help: Replace with `False`
+
+ℹ Suggested fix
+42 42 | if a and "" and False:  # SIM223
+43 43 |     pass
+44 44 | 
+45    |-if a and "foo" and False and "bar":  # SIM223
+   45 |+if a and "foo":  # SIM223
```

Comment:
So in the summary, we mention:

```py
assert ("foo" and False) == "foo"
```

But I don't _think_ this holds, since the left-hand side evaluates to `False`. And in this example here, `a and "foo"` isn't equivalent to `a and "foo" and False and "bar"`, is it?

Reply:
You are right it should check for falsey value not truthy value for `and` operands.
```python
assert ("" and False) == ""
```